shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -329,6 +329,15 @@ fun CompactedJsonLdEntity.toKeyValues(): Map<String, Any> {
     return this.mapValues { (_, value) -> simplifyRepresentation(value) }
 }
 
+fun CompactedJsonLdEntity.replaceEmptyMapsByLists(): Map<String, Any> {
```

Comment:
this is very specific to search service, so it could be left in the search service to keep this one for generic / common purposes